### PR TITLE
Filter Hydrus objects based on a project tag setting

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: app/controllers/hydrus_solr_controller.rb [hunk 1]
```diff
@@ -75,8 +75,7 @@ def delete_from_index
   # Private method to determine if the given object belongs to
   # one of the Hydrus classes.
   def is_hydrus_object(obj)
-    cs = [ Hydrus::AdminPolicyObject, Hydrus::Collection, Hydrus::Item ]
-    return cs.include?(obj.class)
+    obj.tags.include? Settings.hydrus.project_tag
```
Comment (jcoyne) [inline app/controllers/hydrus_solr_controller.rb]: Do we need to handle the case where obj doesn't `respond_to? :tags`?

Comment (cbeer) [inline app/controllers/hydrus_solr_controller.rb]: I think all our objects will have `tags`.

Comment (cbeer) [inline app/controllers/hydrus_solr_controller.rb]: But not, apparently, our test fixtures.

Comment (cbeer) [inline app/controllers/hydrus_solr_controller.rb]: Ok, fixed that, and wrote a test for indexing valid objects.

**Pull request merged**